src/api/app/jobs/project/update_packages_if_dirty.rb

```diff
@@ -0,0 +1,7 @@
+class Project
```

Comment:
Mmmm using Project here could be "dangerous" or misleading??
Isn't better something like ProjectJobs ?